src/gui/rss/rsswidget.h

```diff
@@ -31,9 +31,9 @@
 #ifndef RSSWIDGET_H
 #define RSSWIDGET_H
 
-#include <QPointer>
-#include <QShortcut>
+#include <QWidget>
 
+class QShortcut;
 class ArticleListWidget;
 class FeedListWidget;
 class QListWidgetItem;
```

Comment:
This probably not defined in coding guidelines but I think the grouping rule for headers also applies to forward declarations.
So this would become:
```
class QListWidgetItem;
class QShortcut;
class QTreeWidgetItem;

class ArticleListWidget;
class FeedListWidget;
```

Reply:
I agree but i'll do it in a new commit because i don't want to bother fixing conflicts between the existing ones. i also saw this in some other places and it wouldn't belong to any of the existing commits if the occurrences are many.